module-services/service-time/AlarmOperations.cpp

```diff
@@ -338,6 +333,26 @@ namespace alarms
         }
     }
 
+    bool AlarmOperationsCommon::turnOffAlarmIfFoundInOngoingEvents(std::uint32_t id)
+    {
+        auto found = findSingleEventById(ongoingSingleEvents, id);
+        if (found == ongoingSingleEvents.end()) {
+            return false;
+        }
+
+        switchAlarmExecution(*(*found), false);
+        ongoingSingleEvents.erase(found);
+        processOngoingEvents();
+        handleActiveAlarmsCountChange();
+
+        return true;
+    }
+
+    bool AlarmOperationsCommon::turnOffAlarmIfFoundInBedtime(const uint32_t id)
```

Comment:
It'd be better to use `const std::uint32_t`

Reply:
If function parameters is not a redirection(reference or pointer) so parameters is copied or moved I do prefer do not use const

Reply:
It's true in the case of headers. For implementation, I can't see a reason not to use const.